wpeverest/everest-forms
# Enhancement - Submit form name form by pressing the enter key while adding a new form.

### All Submissions:






### Changes proposed in this Pull Request:

This PR enhances the 'form name' form with submitting by pressing the enter key while adding a new form.

### How to test the changes in this Pull Request:

1. Verify, whether enter key works or not for continuing while adding a new form.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Submit form name form by pressing the enter key while adding a new form.
